static/script/devices/mediaplayer/live/restartable.js

```diff
@@ -56,7 +56,12 @@ require.def(
             },
 
             beginPlayback: function() {
-                this._mediaPlayer.beginPlayback();
+                var config = RuntimeContext.getDevice().getConfig();
+                if (config && config.forceBeginPlaybackToEndOfWindow) {
```

Comment:
Is this really a top level configuration property? The only other config option we support for the media player is under streaming -> overrides. The only other top-level items are _comment and displayQuality (and I don't even know what displayQuality is for).